src/components/ShareTrajectoryModal/index.tsx

```diff
@@ -0,0 +1,206 @@
+        }
+    };
+
+    const modalOptions = {
+        localFile: {
+            content: (
+                <>
+                    <h4>{Warn} The current file is stored on your device.</h4>
+                    <div className={styles.bodyText}>
+                        <h5>
+                            To generate a shareable link, please save the file
+                            in the public cloud using Dropbox, Google Drive, or
+                            Amazon S3 and load the model into Simularium via
+                            URL.
+                            <a
+                                href="https://simularium.allencell.org/tutorial#share-a-link"
+                                target="_blank"
+                                rel="noreferrer"
+                            >
+                                {" "}
+                                Learn more{" "}
+                            </a>
+                        </h5>
+                    </div>
+                </>
+            ),
+            footer: (
+                <Button
+                    className={styles.okButton}
+                    type="default"
+                    onClick={closeModal}
+                >
+                    Ok
+                </Button>
+            ),
+        },
+        networkedFile: {
+            content: (
+                <>
+                    <div>
+                        <Input
+                            className={styles.urlInput}
+                            value={url}
+                            disabled
+                        />
+                        <Button type="text" onClick={copyToClipboard}>
+                            Copy {Link}
+                        </Button>
+                    </div>
+                    <div className={styles.timeInputContainer}>
+                        {" "}
+                        <Checkbox onChange={handleAllowUserInput}></Checkbox>
+                        <p>Start at</p>
+                        <Input
+                            className={styles.timeInput}
+                            disabled={allowTimeInput}
+                            defaultValue={currentTime}
+                            onChange={handleUserInput}
+                        />
+                        <div>
+                            {" "}
+                            /{displayTimes.roundedLastFrameTime}{" "}
+                            {timeUnits ? timeUnits.name : null}{" "}
+                        </div>
+                    </div>
+                </>
+            ),
+            footer: (
+                <Button type="default" onClick={closeModal}>
+                    Close
+                </Button>
+            ),
+        },
+    };
+
+    return (
+        <CustomModal
+            className={classNames(styles.uploadModal, theme.lightTheme)}
+            title="Share Trajectory"
+            width={isLocalFile ? 611 : 550}
```

Comment:
I think Megan left a comment on this too but it got marked as outdated, did UX weigh in on this? Feels odd to have magic numbers here

Reply:
I can check in with @lynwilhelm but yes I pulled these widths from the UX designs. Are you thinking they should be more responsive? In general we have a lot of fixed pixel sizes throughout the CSS in simularium website that should maybe be reviewed down the line in favor or more responsive design.

Reply:
Got it. If this is convention for the project then that's fine :)